src/SmartComponents/Portfolio/Portfolios.js

```diff
@@ -58,24 +58,12 @@ class Portfolios extends Component {
       );
     }
 
-    onClickCreatePortfolio = () => {
-      this.props.showModal({
-        open: true,
-        closeModal: this.props.hideModal
-      }, 'addportfolio');
-
-      this.setState({
-        ...this.state,
-        isOpen: !this.state.isOpen
-      });
-    };
-
     render() {
       let filteredItems = {
         items: this.props.portfolios
         .filter(({ name }) => name.toLowerCase().includes(this.state.filterValue.trim().toLowerCase()))
         .map(item => <PortfolioCard key={ item.id } { ...item } />),
-        isLoading: this.props.isLoading
+        isLoading: this.props.isLoading && this.props.portfolios.lenght === 0
```

Comment:
yes. We basically don't want to show empty screen that is when we should provide something for the user to look at. Once there are some data, we don't need to show placeholders, just update the content.

Reply:
For instance when editing. There is no need to show placeholder just remove the item. Instead of these states: `Portfolios -> Loaders -> Updated portfolios` we can just have: `Portfolios -> Updated portfolios` state transition. Its faster and easier on the users eye (smaller changes on the screen)